src/test/regress/sql/wrong_cancel_message.sql

```diff
@@ -0,0 +1,30 @@
+--- Regression test for this issue:
```

Comment:
we need to add this to a schedule?

Plus, maybe file name could be `statement_cancel_error_message` or such?